docs/topics/api/signing.rst

```diff
@@ -0,0 +1,83 @@
+=======
+Signing
+=======
+
+This API requires :doc:`authentication <auth>`.
+
+-------------------
+Uploading a version
+-------------------
+
+You can upload a new version for signing by issuing a ``PUT`` request
+and including the contents of your add-on in the ``upload`` parameter
+as multi-part formdata. This will create a pending version on the
+add-on and will prevent future submissions to this version unless
+validation or review fails.
+
+Example::
+
+    curl https://addons.mozilla.org/en-US/firefox/api/v3/addons/my-addon/versions/1.0/
+         -XPUT --form 'upload=@build/my-addon.xpi' -H 'Authorization: JWT <jwt-token>'
+
+The response will be the same as the :ref:`check-status` response.
+
+If your upload has the right metadata then it will be submitted for
+validation and you will be able to check its status.
+
+------------------
+Creating an add-on
+------------------
+
+If this is the first time that your add-on's UUID has been seen then
+the add-on will be created as an unlisted add-on when the version is
+uploaded.
+
+.. _check-status:
+
+-----------------------------------
+Checking the status of your upload
+-----------------------------------
+
+You can check the status of your upload by issuing a ``GET`` request.
+There are a few things that will happen once a version is uploaded
+and the status of those events is included in the response.
+
+Once validation is completed (whether it passes or fails) then the
+``processed`` property will be ``true``. You can check if validation
+passed using the ``valid`` property and check the results with
+``validation_results``.
+
+If validation passed then your add-on will be submitted for review.
+In the case of unlisted add-ons this will happen automatically if
+the add-on passes a strict set of tests. If your add-on is listed
+then it will be reviewed by a human and that will take a bit
+longer. Once review is complete then the ``reviewed`` property
```

Comment:
Yeah, if you already have a listed add-on then it will accept a new version and go into the regular review queue.